app/controller/MapController.js

```diff
@@ -38,6 +38,7 @@ Ext.define('CpsiMapview.controller.MapController', {
 
     /**
      * Initialize measure buttons
+     * @param {Ext.button.Button} btn The measure button
```

Comment:
This has nothing to do with this PR, or?

Reply:
No, it doesn't. However, should I create a certain PR for this "minor" change in documentation or should I revert this?

Reply:
If easily done, then yes. Don't let this block you though.